src/system/ui/pages/post/index.js

```diff
@@ -204,6 +204,15 @@ export const Post = ( { location, match, history } ) => {
 							value: item.date
 
 						},
+						post_author: {
+							label: __( 'Author' ),
+							labelPlacement: 'beside',
+							component: 'select',
+							id: 'post_author',
+							options: item.authorList,
+							value: item.post_author,
+							isVisible: isAdmin ? true : false
```

Comment:
@shahidajmeri786 this isn't showing up for me as `isAdmin` is an empty string, even though I'm logged in as an admin. Can you look into that?

Reply:
Fixed the issue.